deployment/manager/app/manager_config.json

```diff
@@ -6,13 +6,13 @@
     },
     "smartcity": {
       "styles": ":host > * {--or-app-color2: #F6F7FA; --or-app-color3: #1B1464; --or-app-color4: #662D91;}",
-      "logo": "/images/logo_smartcity.svg",
-      "logoMobile": "/images/logo-mobile_smartcity.svg"
+      "logo": "api/master/configuration/manager/image/logo_smartcity.svg",
```

Comment:
It's ok to change the paths for new config but we have configs already deployed on instances which will be using `/images/...` paths. When the manager config is saved the images should be copied to the storageDir() also otherwise the images could still be lost on redeploy.

Reply:
@richturner I was thinking of performing a test when updating the manager_config file. Basically, check that the files referenced as images in the config are retrievable through the storageDir, and if they're not, check if they are retrievable from the deployment path. If that happens, copy them into the storageDir. I can't think of a more efficient way, any other ideas?

Reply:
@richturner refer to `ConfigurationService#checkAndFixImageReferences`